CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+0.11.0 / 2020-12-08
+===================
+
+* **[Feature]**: Add the completion of the keyword `bool` (#89)
+* **[Feature]**: Add a function `enricher` that can be used to enrich the completion with a custom one
```

Comment:
Is this a duplicate of the point below ("Allow the possibility to inject a custom `CompleteStrategy`")?

Reply:
ah no it's two different things. `enricher` won't override the default completion strategy, but you can implement your own `CompletionStrategy` and do not use the default one if you want